docs/docusaurus.config.js

```diff
@@ -16,56 +18,92 @@ const legacyVersion = {
   to: 'https://legacy-docs-v1.rasa.com',
 };
 
+// FIXME: when deploying this for real, change to '/docs/rasa/'
+const BASE_URL = '/docs/rasa/next/';
+const SITE_URL = 'https://rasa.com';
+
 module.exports = {
   title: 'Rasa Open Source Documentation',
   tagline: 'Rasa Open Source Documentation',
-  url: 'https://rasa.com',
-  // FIXME: when deploying this for real, change to '/docs/rasa/'
-  baseUrl: '/docs/rasa/next/',
+  url: SITE_URL,
+  baseUrl: BASE_URL,
   favicon: 'img/favicon.ico',
   organizationName: 'RasaHQ',
   projectName: 'rasa',
   themeConfig: {
+    colorMode: {
+      defaultMode: 'light',
+      disableSwitch: true,
+    },
+    versions: {
+      label: 'Versions',
+      to: '/', // "fake" link
+      position: 'left',
+      items:
+        versions.length > 0
+          ? [
+              {
+                label: versions[0],
+                to: '/',
+                activeBaseRegex: versions[0],
+              },
+              ...versions.slice(1).map((version) => ({
+                label: version,
+                to: `${version}/`,
+                activeBaseRegex: version,
+              })),
+              {
+                label: 'Master/Unreleased',
+                to: 'next/',
+                activeBaseRegex: `next`,
+              },
+              legacyVersion,
+            ]
+          : [
+              {
+                label: 'Master/Unreleased',
+                to: '/',
+                activeBaseRegex: `/`,
+              },
+              legacyVersion,
+            ],
+    },
     navbar: {
-      title: 'Rasa Open Source',
+      hideOnScroll: false,
+      title: 'Rasa Docs',
       logo: {
         alt: 'Rasa',
-        src: 'https://rasa.com/static/60e441f8eadef13bea0cc790c8cf188b/rasa-logo.svg',
+        src: `${SITE_URL}/static/60e441f8eadef13bea0cc790c8cf188b/rasa-logo.svg`,
```

Comment:
should we use `path.join()` here too?

Reply:
I'll kill this whole line eventually, it's not stable to rely on hashed assets generated by gatsby, I'll bring `rasa-logo.svg` in to `/static/`

Reply:
I'll do it in the next one, if that's OK. It will be internalized in the theme.